### DISCUSSION TITLE
Space ROS Technical Meeting Agenda 2024-10-31

### DISCUSSION THREAD

**Discussion by Bckempa** [space-ros/space-ros#224] [General]
Previous Agenda: https://github.com/space-ros/space-ros/discussions/223
Active Milestone: [`jazzy-2024.10.0`](https://github.com/orgs/space-ros/projects/3/views/11)

It's release day, the spookiest day of the year!

### Agenda
- [ ] Introduce any new attendees.
- [ ] Summarize any PRs merged since last meeting.
      - https://github.com/space-ros/check-commit-message-action/pull/4
- [ ] Issue Review 
- [ ] Open next meeting discussion.

**Comment by Bckempa** [comment]
Release Status:

ToDo - 5
In Progress/Review - 15
Done - 39

- Jazzy Support Status: https://github.com/space-ros/space-ros/issues/157#issuecomment-2447721024
- https://github.com/space-ros/docker/pull/141 blocked by https://github.com/space-ros/space-ros/pull/145, are these ready for review?
- https://github.com/space-ros/space-ros/issues/218 stalled on `ament` org, do we need to fork?
- https://github.com/space-ros/docker/issues/197 Design question.
- @dsobek, release working session?

**Reply by Bckempa** [reply]
There was discomfort about doing all these merges at the last day before the release. The consensus during the meeting was to complete the current release on humble but move all further development on jazzy immediately.

1. Follow the regular release process for the current, Humble based main branch.
2. Merge all the Jazzy related changes shortly after so they will be live on the `main` tag as a release candidate.
3. Dogfood the Jazzy release for a cycle, putting the January release on Jazzy.

**Reply by xfiderek** [reply]
https://github.com/space-ros/docker/pull/141 and https://github.com/space-ros/space-ros/pull/145 from the second bulletpoint are not ready. I have not had time to rework that after migration of main Earthfile from docker to spaceros repo. I hope to address it soon. Meanwhile I'll change status of them to draft to avoid confusion.

**Reply by asimonov** [reply]
#218 - we agreed we should aim to slowly merge [some] of 22 commits on `spaceros` branch of `ament_lint` to upstream and finally switch to `rolling` version. implementation depends on Ivan's ability to create branches/tags on `ament_lint`...

in any case it should not be in 'humble' based release

**Comment by Bckempa** [comment]
Next agenda - https://github.com/space-ros/space-ros/discussions/233
